src/react-apps/applications/runtime/package.json

```diff
@@ -82,6 +82,7 @@
     "node-sass": "4.11.0",
     "prettier": "1.16.4",
     "prettier-tslint": "0.4.2",
+    "react-content-loader": "^4.2.1",
```

Comment:
Should we not use specific versions?

Reply:
I'm still conflicted by this. The `^` says to take the lastest of a certain major-patch.

If there are breaking changes in a minor or patch version of a package, the package-maintainers are doing things wrong. We can't failsafe for everything. It's like saying the app owners should have fire extinguishers because it might burn in their building.

Reply:
My little rant is over. And will remove the `^`.